### ISSUE TITLE
1 video always remaining

### ISSUE THREAD

**Issue by atropos112** [tubearchivist/tubearchivist#253]
Hello there, 

I've been self hosting tubearchivist via docker for over a week now and I have to say it works very well, does pretty much everything I would like it to.

I do however persistently see a problem occurring, after (auto)scanning and having it download (via the "Start Download") button, there is always a video that does not dowload, even after pressing "Dowload Now" on the video specifically.

This is not video specific as when a next video is scanned in the video I couldn't download before will now download and the most recent video will be 'un-downloadable'. Any idea how I can fix this ?

**Comment by bbilly1** [comment]
Please enable debugging with the environment variable `DJANGO_DEBUG=TRUE` and provide docker logs.

**Comment by atropos112** [comment]
When pressing the "Download Now" button this is what I get 
```
TubeArchivist-rzr        | dlofjA3Y3aU: downloading now
TubeArchivist-rzr        | set task id: f5f62413-5829-42c5-9a6c-9685598792f1
TubeArchivist-rzr        | [pid: 47|app: 0|req: 21/21] 172.24.0.27 () {90 vars in 1653 bytes} [Tue May 24 05:32:07 2022] POST /process/ => generated 17 bytes in 48 msecs (HTTP/1.1 200) 7 headers in 220 bytes (1 switches on core 0)
TubeArchivist-rzr        | [2022-05-24 05:32:07,369: INFO/MainProcess] Task home.tasks.download_single[f5f62413-5829-42c5-9a6c-9685598792f1] received
TubeArchivist-rzr        | [2022-05-24 05:32:07,372: WARNING/ForkPoolWorker-16] Added to queue with priority: dlofjA3Y3aU
TubeArchivist-rzr        | [2022-05-24 05:32:07,611: WARNING/ForkPoolWorker-16] auto delete older than 1 days
TubeArchivist-rzr        | [2022-05-24 05:32:07,630: WARNING/ForkPoolWorker-16] UCBJycsmduvYEL83R_U4JriQ: delete older than 1d
TubeArchivist-rzr        | [2022-05-24 05:32:07,649: INFO/ForkPoolWorker-16] Task home.tasks.download_single[f5f62413-5829-42c5-9a6c-9685598792f1] succeeded in 0.27882652200059965s: None
TubeArchivist-rzr        | [pid: 47|app: 0|req: 22/22] 172.24.0.27 () {82 vars in 1447 bytes} [Tue May 24 05:32:07 2022] GET /progress/ => generated 122 bytes in 5 msecs (HTTP/1.1 200) 7 headers in 221 bytes (1 switches on core 0)
TubeArchivist-rzr        | [pid: 47|app: 0|req: 23/23] 172.24.0.27 () {86 vars in 1593 bytes} [Tue May 24 05:32:07 2022] GET /static/img/icon-close.svg => generated 2707 bytes in 1 msecs via sendfile() (HTTP/1.1 200) 10 headers in 322 bytes (0 switches on core 0)
TubeArchivist-rzr        | [pid: 47|app: 0|req: 24/24] 172.24.0.27 () {86 vars in 1593 bytes} [Tue May 24 05:32:07 2022] GET /static/img/icon-stop.svg => generated 2080 bytes in 1 msecs via sendfile() (HTTP/1.1 200) 10 headers in 322 bytes (0 switches on core 0)
TubeArchivist-rzr        | [pid: 47|app: 0|req: 25/25] 172.24.0.27 () {82 vars in 1447 bytes} [Tue May 24 05:32:10 2022] GET /progress/ => generated 122 bytes in 3 msecs (HTTP/1.1 200) 7 headers in 221 bytes (1 switches on core 0)
TubeArchivist-rzr        | [pid: 47|app: 0|req: 26/26] 172.24.0.27 () {82 vars in 1447 bytes} [Tue May 24 05:32:13 2022] GET /progress/ => generated 122 bytes in 9 msecs (HTTP/1.1 200) 7 headers in 221 bytes (1 switches on core 0)
TubeArchivist-rzr        | [pid: 47|app: 0|req: 27/27] 172.24.0.27 () {82 vars in 1447 bytes} [Tue May 24 05:32:16 2022] GET /progress/ => generated 122 bytes in 4 msecs (HTTP/1.1 200) 7 headers in 221 bytes (1 switches on core 0)
TubeArchivist-rzr        | [pid: 47|app: 0|req: 28/28] 172.24.0.27 () {82 vars in 1447 bytes} [Tue May 24 05:32:19 2022] GET /progress/ => generated 122 bytes in 4 msecs (HTTP/1.1 200) 7 headers in 221 bytes (1 switches on core 0)
TubeArchivist-rzr        | [pid: 47|app: 0|req: 29/29] 172.24.0.27 () {82 vars in 1447 bytes} [Tue May 24 05:32:22 2022] GET /progress/ => generated 122 bytes in 14 msecs (HTTP/1.1 200) 7 headers in 221 bytes (1 switches on core 0)
TubeArchivist-rzr        | [pid: 47|app: 0|req: 30/30] 172.24.0.27 () {82 vars in 1447 bytes} [Tue May 24 05:32:25 2022] GET /progress/ => generated 122 bytes in 4 msecs (HTTP/1.1 200) 7 headers in 221 bytes (1 switches on core 0)
TubeArchivist-rzr        | [pid: 47|app: 0|req: 31/31] 172.24.0.27 () {82 vars in 1447 bytes} [Tue May 24 05:32:29 2022] GET /progress/ => generated 16 bytes in 10 msecs (HTTP/1.1 200) 7 headers in 220 bytes (1 switches on core 0)
TubeArchivist-rzr        | [pid: 47|app: 0|req: 32/32] 172.24.0.27 () {84 vars in 1563 bytes} [Tue May 24 05:32:29 2022] GET /downloads/ => generated 7227 bytes in 61 msecs (HTTP/1.1 200) 8 headers in 398 bytes (1 switches on core 0)
TubeArchivist-rzr        | [pid: 47|app: 0|req: 33/33] 172.24.0.27 () {82 vars in 1447 bytes} [Tue May 24 05:32:29 2022] GET /progress/ => generated 16 bytes in 9 msecs (HTTP/1.1 200) 7 headers in 220 bytes (1 switches on core 0)
```

Here TubeArchivist-rzr is the name of the container.

**Comment by bbilly1** [comment]
Does that problem persist if you deactivate the *delete older than x days* functionality?

**Comment by atropos112** [comment]
It does not, thank you. This is not the intended behavior is it ?

**Comment by bbilly1** [comment]
No, that's not the intended behavior. But I can't reproduce that. For me it always downloads the complete queue, even with the auto delete setting activated. 

Anybody else can confirm that issue?

If you want to further debug that, please reach out on Discord.

**Comment by lamusmaser** [comment]
This was happening with 0.1.5, but it looks like it might be resolved in 0.1.6. @atropos112 - can you confirm if you are seeing it after upgrade?

**Comment by atropos112** [comment]
Just checked, it works well now even with 1 day delete window, thank you !
